packages/salesforcedx-vscode-core/src/messages/i18n.ts

```diff
@@ -46,7 +46,11 @@ export const messages = {
     'SFDX: Authorize an Org using Session ID',
   force_auth_access_token_login_bad_oauth_token_message:
     'The session ID that you are trying to use is not valid. Check if it has expired, or use a valid session ID.',
-
+  force_auth_web_login_device_code_parse_error:
+    'There was an unexpected error authorizing to your org in a container environment.',
+  force_auth_device_login_enter_code:
+    'Enter %s user code in the verification URL %s',
+  action_required: '=== Action Required!',
```

Comment:
@sbudhirajadoc  I think only the above messages are new. These are show to the user in the log output. See https://github.com/forcedotcom/salesforcedx-vscode/pull/4077/files#r865412273 for example

Reply:
"**There was an unexpected error authorizing to your org in a container environment.**'" isn't very a helpful error message. I would like to add more context that would be helpful to a user. Let's work on this  together next week, @jonnyhork .Thanks, @gbockus-sf !